Extensions/audio_plus.js

```diff
@@ -100,116 +94,49 @@ XKit.extensions.audio_plus = {
 		var audio_plus = XKit.extensions.audio_plus;
 		var controls = audio_plus.pop_out_controls;
 		if (controls.classList.contains("playing")) {
-			audio_plus.stop_current_instance();
+			audio_plus.current_player.querySelector('audio').pause();
```

Comment:
Would it be too annoying to register this handler when we start playing, so we can use closures instead of this ugly instance variable?